.gitignore

```diff
@@ -118,6 +118,7 @@ classification/data/*
 data/
 !contrib/action_recognition/r2p1d/**
 !contrib/crowd_counting/crowdcounting/data/
+!scenarios/action_recognition/data
```

Comment:
General comments:
- Could you please also add a readme, similar to image classification or object detection. And then mention that this is work-in-progress (see here how to write in green text: https://github.com/Azure/ObjectDetectionUsingCntk)
- The .txt and files in your local data directory, can they please be moved to ROOT/data/misc/ e.g.
ROOT/data/misc/hmdb51_vid_train_split_1.txt
ROOT/data/misc/kinetics400_label_map.txt
- Can we avoid this file since it's nearly empty?: scenarios/action_recognition/data/samples/label.txt
- Can you also please move over the annotation scripts from \contrib\action_recognition\video_annotation to \scenarios\action_recognition\annotation
- Please remove the mp4 file from here, and instead download it from an url I can create for you

Reply:
- Readme added with note 'work-in-progress'
- The .txt files have been moved to ROOT/data/misc/hmdb51 and ROOT/data/misc/kinetics400. hmdb51 and kinetics are separate to make the folder structure clear.
- The file scenarios/action_recognition/data/samples/label.txt has been removed and notebook updated accordingly
- The mp4 file has been removed and the notebook has been updated accordingly to use the url link you provided.

Reply:
I'll move the annotation scripts after doing some testing. Will do that in a separate PR.